calculated_policies/redshift.tf

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
What is this file?

Reply:
this is my notes file, shouldn't have pushed, will remove